db/migrate/20241204111437_cleanup_invalid_partner_profiles.rb

```diff
@@ -0,0 +1,61 @@
+class CleanupInvalidPartnerProfiles < ActiveRecord::Migration[7.1]
+  def up
+    # ActiveRecord::Base.logger = Logger.new(STDOUT)
+    invalid_profiles = Partners::Profile.all.reject(&:valid?)
```

Comment:
One other question. Are there many `Partners::Profile` records in production? If there are many, then loading them all could cause a memory spike and fire off thousands of database queries.

The safer option when dealing with lots of records would be something like:
```ruby
Partners::Profile.find_each(batch_size: SOME_NUMBER) do |profile|
  next if profile.valid?
  # code
  sleep(0.01) # throttle
end
```
If there aren't many records it doesn't really matter and ignore this comment.

Reply:
Define many *g*.  There are about 6000.

Reply:
find\_each is technically better, but for a one-off migration like this it's usually not worth the fuss.